overlord/devicestate/handlers_test.go

```diff
@@ -89,6 +91,7 @@ func (s *deviceMgrSuite) TestSetModelHandlerSameRevisionNoError(c *C) {
 
 	t := s.state.NewTask("set-model", "set-model test")
 	chg := s.state.NewChange("dummy", "...")
+	chg.Set("new-model", string(asserts.Encode(model)))
 	chg.AddTask(t)
 	chg.Set("new-model", string(asserts.Encode(model)))
```

Comment:
AFAICT we have `chg.Set("new-model", string(asserts.Encode(model)))` twice now - can we remove this line?

Reply:
yes, probably some merge artifact